packages/server/email/components/SummaryEmail/ExportToCSV.tsx

```diff
@@ -33,10 +31,34 @@ const query = graphql`
         }
         endedAt
         ... on ActionMeeting {
+          agendaItems {
```

Comment:
I created a local fragment in ``ExportToCSV``. I didn't use ``DiscussionThread`` directly here because we need to access to really deep level of data here. If we use ``DiscussionThread`` we have to unmask all of the child fragments.